### PR TITLE
Fixed inconsistent tiles, rebalanced explosives

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
This removes beeping from the grenade and minibomb, since currently due to explosion jank their yields are too low for anything useful but people. There's been quite a bit of discussion in Discord about this; as long as they can't breach removing beeping should be fine. You need to land a direct hit with either to crit someone in armor, and it's pretty hard to time.

In addition, the cryobeaker menace was nerfed by reducing the capacity from 60 to 40.

**Changelog**

:cl: Rane
- fix: Wood floor tiles can no longer be placed directly on space, and do not space so easily.
- tweak: Cryobeaker capacity reduced from 60 to 40.
- tweak: Explosive grenades and minibombs no longer beep.

### CODE REVIEW

File: Resources/Prototypes/Tiles/floors.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
-﻿- type: tile
+
+- type: tile
   id: floor_steel
   name: steel floor
   texture: "steel"
   base_turfs:
+  - space
   - plating
```
Comment (Zumorica) [inline Resources/Prototypes/Tiles/floors.yml:8]: This is a *terrible* solution. It has the side effect of being able to build steel floors on space tiles directly.
This bug needs to be fixed by addressing the explosion code, not by changing this.

This also goes for every other `base_turfs` you changed

**Review by Zumorica — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Tiles/floors.yml (hunk 1 above)
Comment (Elijahrane) [reply Resources/Prototypes/Tiles/floors.yml:8]: Moony told me to do exactly this

Comment (moonheart08) [reply Resources/Prototypes/Tiles/floors.yml:8]: Yea so this tells me explosion code and construction code disagree on what base_turfs is supposed to do.

Comment (metalgearsloth) [reply Resources/Prototypes/Tiles/floors.yml]: Explosion code is from like 4 years ago so obviously out of date.

**Pull request merged**